### PR TITLE
CI status on README; install instructions

### PR DESCRIPTION
**Pull request by whoburg**
Final cleanup to close out #509 

 - show Jenkins status for all three machines on README
 - eliminate duplicate debugging info from install instructions
 - update debugging info with other potential failure modes as suggested by @galbramc

### CODE REVIEW

**Comment by whoburg** [comment]
@bqpd : ready for review.

File: docs/source/installation.rst [hunk 1]
```diff
@@ -125,23 +99,25 @@ Windows
...
+  - You install a new solver (mosek or cvxopt) after installing GPkit
+  - You delete the ``.gpkit`` folder from your home directory
+  - You see a ``Could not load settings file.`` message
+  - You see a ``Could not load MOSEK library: ImportError('$HOME/.gpkit/expopt.so not found.`` error.
+To rebuild GPkit, do the following:
+  - Start ``ipython``
+  - Run the command ``import os; os.chdir(os.path.dirname(gpkit.__file__)); del gpkit; import build; build.build_gpkit()``
+  - Exit ipython
+  - Run ``python -c "import gpkit.tests; gpkit.tests.run()"``
+  - If any tests fail, email ``gpkit@mit.edu``.
```
Comment (whoburg) [inline docs/source/installation.rst]: @galbramc says,
```
An alternative way of forcing the build is to run the commands:

pip uninstall gpkit
pip install --no-cache-dir --no-deps gpkit

You could suggest that to users if they have the missing settings or .gpkit, or install a new solver.
```
do we like that method, or the one above, better?

**Comment by galbramc** [comment]
So the failure here was surprising as documentation updated, but it looks like you are testing how long something takes to run. I'm guessing the machine was busy so the execution time was too long

```
  File "/Users/jenkins/workspace/gpkit_PullRequest_Unit/buildnode/macys/optimizer/mosek/gpkit/tests/t_solution_array.py", line 52, in test_call_time
    self.assertTrue(time.time() - t1 <= 0.05)
AssertionError: False is not true
```

Also, if you use assertLess rather than assertTrue, we would know what the delta time actually was rather than just know it was a failure...

**Comment by galbramc** [comment]
test this please

File: docs/source/installation.rst (hunk 1 above)
Comment (bqpd) [inline docs/source/installation.rst]: I like the one above better, pushed.

**Comment by bqpd** [comment]
Ready to merge.

**Comment by whoburg** [comment]
test this please

**Comment by whoburg** [comment]
hmph we're getting sporadic failures on that timing test. Macys ElCap seems to take consistently longer than any other machine to run the unit tests.

**Comment by whoburg** [comment]
created #567 for the sporadic timing test failure

**Pull request merged**